### PR TITLE
Link to kiosk

### LINKED ISSUE #109: Link to Kiosk Mode On Event Page
We need to be able to link to the kiosk for a specified event from the Track Volunteers tab on an event's page.
image
image

This should be implemented somewhere in the area of the search bar above the table for adding volunteers.
Related files:
- _app/controllers/events/routes.py_
- _app/controllers/admin/routes.py_
- _app/templates/events/eventKiosk.html_
- _app/templates/events/trackVolunteers.html_
- _app/static/trackVolunteers.js_

### PR DESCRIPTION
**Pull request by mupotsal**
This PR fixes issue #109. We need a link to the Kiosk page from the Track volunteer Page for an admin/Celts student staff/Celts staff . We have implemented it as a button. 
Screen Shot 2021-09-10 at 8 46 44 AM

Test. Login as a Celts admin or faculty or Celts student staff and navigate to track Volunteers Page (program/event/track_volunteers).

### CODE REVIEW

File: app/templates/events/trackVolunteers.html [hunk 1]
```diff
@@ -30,13 +30,23 @@ <h3 align="center">{{event.eventName}}</h3><br>
...
+              <div><button type="button" class="btn btn-secondary btn-sm" data-bs-toggle="modal" data-bs-target="#addVolunteerModal">
+                <i class="bi bi-plus-circle"></i>
+              </button></div>
+            </div>
+          </div>
+          <div class='form-group d-inline-flex' style="padding: 0 0 0 2rem">
+            <a class="btn btn-info" href= "/{{event.id}}/kiosk" role="button">Kiosk Scanner</a>
+          </div>
+        </div>
+      </div>
```
Comment (BrianRamsay) [inline app/templates/events/trackVolunteers.html]: Is it necessary to have this many nested divs to get the desired behavior?

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
I think 'Kiosk Entry' is a better name than Kiosk Scanner. 

We probably also want to look at the UI options for entry right now. Having the Search, the +, and Kiosk button all next to each other is confusing to me. But we can deal with that in another issue.

File: app/templates/events/trackVolunteers.html (hunk 1 above)
Comment (sreynit02) [reply app/templates/events/trackVolunteers.html]: No we don't :). We got rid of two divs and it still works

**Comment by sreynit02** [comment]
PR updated.

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**